hub2labhook/api/hook.py

```diff
@@ -90,8 +90,10 @@ def github_event():
          params['action'] not in ['opened', 'reopened', 'synchronize'])):
         return jsonify({'ignored': True})
 
-    task = tasks.pipeline.s(params, dict(request.headers.to_list()))
+    headers = dict(request.headers.to_list())
+    task = tasks.pipeline.s(params, headers)
     task.link(tasks.update_github_statuses.s())
+    task.link_error(tasks.update_github_statuses_failure.s(params, headers))
```

Comment:
what's link_error ?

Reply:
Similar to `link()`, [adds an option][1] to trigger another task, specifically when an error occurs during execution of the preceding task.


[1]: https://github.com/celery/celery/blob/master/celery/canvas.py#L342